lib/manageiq/providers/oracle_cloud/engine.rb

```diff
@@ -6,6 +6,11 @@ class Engine < ::Rails::Engine
 
         config.autoload_paths << root.join('lib').to_s
 
+        initializer :append_secrets do |app|
```

Comment:
You _might_ have to have some sort of `:before => :something_rails_secrets` here to make sure it slots into the right initialization order.

Reply:
Would be cool if we could find a way to "auto-inject" this into every provider

Reply:
> You might have to have some sort of :before => :something_rails_secrets here to make sure it slots into the right initialization order.

I think this is fine...I can't find anything in the default initializers where we would hook it differently.